io.catenax.part_as_planned/2.0.0/metadata.json

```diff
@@ -0,0 +1 @@
+{ "status" : "deprecate"}
```

Comment:
I think you want to release and not deprecate it. please change the status to release.